OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3538,91 +3545,244 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <decoNote xml:id="b2" type="bindingMaterial">
                                     <material key="wood"/>
                                     <material key="leather"/>
                                     <material key="textile"/>
                                     <material key="wood"/>
                                 </decoNote>
-                             
-                          
+                                <decoNote xml:id="b3" type="Cover" color="reddish brown"
+                                    >Blind-tooling decoration on the full leather cover including
+                                    turn-ins, edges, and spine. The decoration was carefully applied
+                                    using different tools.</decoNote>
                                 <decoNote xml:id="b4" type="SewingStations">4</decoNote>
                             </binding>
                         </bindingDesc>
                           
                         
                     </physDesc>
                     
-                <!--    <history>
+                    <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <!-- Dating probably to be revised when taking miniatures into account
+                                (second quarter of 18th century dating by Juel Jensen, probably mainly from analysis of the miniatures),
+                            I cannot restrict the handwriting to such a narrow range-->
+                            The manuscript can be dated palaeographically to the 
+                            <origDate notBefore="1600" notAfter="1799" evidence="lettering">17th-18th century</origDate>. 
+                            Several persons are mentioned in supplication formulas throughout the manuscript:
+                            <persName ref="PRS12911KidanaK"/> and, less often,  <persName ref="PRS12964Gorgoryos"/> and
+                            <persName ref="PRS13026KenfaM"/>.
+           <!--I had created <persName ref="PRS13031WaldaA"/> before looking at the miniatures, he must be mentioned somewhere.
+                            But I did not find him again in the text :(-->
+                                <persName ref="PRS12911KidanaK"/> is also mentioned in the <ref target="#coloph1">colophon</ref> as the commissioner 
+                                and donor of the manuscript.
+                                Miniatures depict <persName ref="PRS12911KidanaK"/> <!--(<ref target="#d"/>)-->, 
+                            <persName ref="PRS13031WaldaA"/> <!--(<ref target="#d"/>)-->
+                            and <persName ref="PRS12964Gorgoryos"/> <!--(<ref target="#d"/>)--> in the pose of donors.
+                                The church identified as Madḫāne ʿĀlam in the colophon and inventory is unlikely to be <ref target="INS0101MadhaneAlam"/>, 
```

Comment:
The statement is not clear to me.

Reply:
Okay, good! Makes sense :)

Reply:
Perhaps this requires a bit more info for readers who won't be familiar with the topic. e.g. The colophon and inventory state that the manuscript was part of the property of a church dedicated to the Saviour of the World (Madḫāne ʿĀlam). However, it is difficult to determine to which church these notes refer to, since many Ethiopian churches are named in his honour -- including  <ref target="INS0101MadhaneAlam"/>, from where the manuscript was likely taken in 1868, and <ref target="XXXGondarMadhaneAlam"/> in Gondar, from where <Tewodros> took many manuscripts for his own capital.